recipes/codespell/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "codespell" %}
+{% set version = "1.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8f1bc15ca4322b72213b1cf13eb437b3c3956e0f423ab414d294d7e77e0e4130
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+          - codespell = codespell_lib:_script_main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - codespell --version
+
+about:
+  home: https://github.com/codespell-project/codespell
+  license: GPL-2.0
+  license_family: GPL
```

GPL-2.0 requires the licence text to be included with all copies of the software:
```suggestion
  license_family: GPL
  license_file: LICENSE
```
If it's not in the tarball you can download/include it in the recipe directory.